coderd/notifications/types/payload.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+// MessagePayload describes the JSON payload to be stored alongside the notification message, which specifies all of its
+// metadata, labels, and routing information.
+//
+// Any BC-incompatible changes must bump the version, and special handling must be put in place to unmarshal multiple versions.
+type MessagePayload struct {
+	Version string `json:"_version"`
```

Comment:
nit: is there a reason we prefix this with an underscore?

Reply:
I wanted to indicate that this was metadata.